packages/atomic/src/components.d.ts

```diff
@@ -213,6 +213,40 @@ export namespace Components {
          */
         "rangeAlgorithm": RangeFacetRangeAlgorithm;
     }
+    interface AtomicNumericFacetV1 {
+        /**
+          * Whether to display the facet values as checkboxes (multiple selection) or links (single selection). Possible values are 'checkbox' and 'link'.
+         */
+        "displayValuesAs": 'checkbox' | 'link';
+        /**
+          * Specifies a unique identifier for the facet.
+         */
+        "facetId"?: string;
+        /**
+          * The field whose values you want to display in the facet.
+         */
+        "field": string;
+        /**
+          * The non-localized label for the facet.
+         */
+        "label": string;
+        /**
+          * The number of values to request for this facet, when there are no manual ranges.
+         */
+        "numberOfValues": number;
+        /**
+          * The algorithm that's used for generating the ranges of this facet when they aren't manually defined. The default value of `"equiprobable"` generates facet ranges which vary in size but have a more balanced number of results within each range. The value of `"even"` generates equally sized facet ranges across all of the results.
+         */
+        "rangeAlgorithm": RangeFacetRangeAlgorithm;
```

Comment:
Curious to know what happens for documentation when we reference types from headless, how it will end up appearing.

Reply:
Handwritten docs will be made unless something is implemented to generate them. It would depend largely on how many Headless type references will end up in the Atomic docs.

Reply:
Parses it really well, here's a snippet from the json
![Screen Shot 2021-06-17 at 10 15 22 AM](https://user-images.githubusercontent.com/4923043/122414167-ef10f300-cf54-11eb-898d-174b4cdb6f3c.png)

Reply:
Super! This is coming from Stencil's generated json?

Reply:
Yes!